src/utils/data-scale-utils.js

```diff
@@ -62,3 +62,15 @@ export function getLinearDomain(data, valueAccessor) {
 
   return range.map((d, i) => (d === undefined ? i : d));
 }
+
+/**
+ * return linear domain for an array of data. A log scale domain cannot contain 0
+ * @param {Array} data
+ * @param {function} valueAccessor
+ * @returns {Array} domain
+ */
+export function getLogDomain(data, valueAccessor) {
```

Comment:
This is definitely a hack. Should we wrap the log scale instead?

Reply:
No I think this is fine